mobify/platform-scaffold
# Fix SFCC / stub IM result import paths

# Fix SFCC / stub IM result import paths

 **JIRA**: N/A

## Changes
- Some of the import paths on non-merlin's connectors were pointing at the IM folder which is no longer in the scaffold repo

## How to test-drive this PR
- Use SFCC connector
- Run `npm run dev`
- Ensure there are no `Cannot resolve module` webpack errors
- Test Merlin's Potions using the SFCC connector
